Src/ReleaseHistory.md

```diff
@@ -16,6 +16,10 @@
 - UER => eliminate unhandled exceptions in rules
 - UEE => eliminate unhandled exceptions in engine
 
+## Unreleased
+
+- BUG: Resolve unhandled `OutOfMemory` and `NullReferenceException` exceptions when analyzing folders that contain big files even when using --file-size-in-kb. [#621](https://github.com/microsoft/sarif-pattern-matcher/pull/621)
```

Comment:
>Resolve [](http://example.com/codeflow?start=7&length=7)



Resolve `OutofMemoryException` and `NullReferenceException' failures resulting from a failure to honor file size scan limits set by `--file-size-in-kb` argument.